resources/schemas.json

```diff
@@ -792,17 +792,6 @@
                 },
                 "title": "AddFileResultView",
                 "description": "The result of adding a file to a LOC"
-            },
-            "VoidLocView": {
-                "type": "object",
-                "properties": {
-                    "reason": {
-                        "type": "string",
-                        "description": "The voiding reason"
-                    }
-                },
-                "title": "VoidLocView",
-                "description": "The parameters of LOC voiding"
```

Comment:
Why is this view removed ?
If you do `yarn run generate-views` you cannot compile any more because `VoidLocView` is still used in the controller.

Reply:
Indeed, my bad. Fixing that right now.

Reply:
Done